README.md

```diff
@@ -48,63 +59,125 @@ python scripts/make.py "qt/bin" --lib-path "nomacs/3rd-party/build"
 - check if your Qt is set correctly (otherwise set the path to `qt_install_dir/qtbase/bin/qmake.exe`)
 - check if your builds proceeded correctly
 
-## Build nomacs (Ubuntu)
+## Build nomacs (Linux/Unix)
 
-Get the required packages:
+Before you build nomacs, please note the following:
 
-- For Ubuntu 21.04 and above:
+- We recommend using the Qt6 version of nomacs. However, you may want to match the Qt version of your desktop environment (e.g. KDE in Ubuntu 24.04 is Qt5-based).
+- [kimageformats-plugins](https://github.com/KDE/kimageformats) is an optional dependency that provides additional formats such as AVIF and HEIC/HEIF. The Qt version of the plugins should match the Qt version when compiling nomacs.
```

Comment:
The original repo is https://invent.kde.org/frameworks/kimageformats

You can mention also JPEG XL (JXL) format which is also supported by kimageformats.